ui/packages/consul-ui/app/models/auth-method.js

```diff
@@ -0,0 +1,24 @@
+import Model, { attr } from '@ember-data/model';
+
+export const PRIMARY_KEY = 'uid';
+export const SLUG_KEY = 'Name';
+
+export default class AuthMethod extends Model {
+  @attr('string') uid;
+  @attr('string') Name;
+
+  @attr('string') Datacenter;
+  @attr('string') Namespace;
+  @attr('string', { defaultValue: () => '' }) Description;
+  @attr('string', { defaultValue: () => '' }) DisplayName;
+  @attr('string', { defaultValue: () => '' }) TokenLocality;
```

Comment:
I think the API docs say that if this is unset then `local` is assumed, so I'd probably default this to `local`.